packages/lab/src/list-next/ListItemNext.tsx

```diff
@@ -106,9 +106,9 @@ export const ListItemNext = forwardRef<HTMLLIElement, ListItemNextProps>(
     return isOverflowed ? (
       <Tooltip
         disabled={!isOverflowed}
-        open={focused}
         content={content}
         hideIcon
+        open={focused || undefined}
```

Comment:
not sure if this is the best fix for this one, would expect focused to be well defined regardless of mouse or keyboard but im unsure without looking too deeply